torchrl/modules/models/models.py

```diff
@@ -1034,3 +1035,90 @@ def forward(
 
         input = self.mlp(input)
         return self._lstm(input, hidden0_in, hidden1_in)
+
+
+class GRUNet(nn.Module):
+    """
+    The GRUNet is a neural network composed of a GRU layer encapsulated
+    between two MLPs.
+
+    Args:
+        mlp_input_kwargs (dict):  kwargs for the MLP before the GRU
+        mlp_output_kwargs (dict): kwargs for the MLP after the GRU
+        gru_kwargs (dict): kwargs for the GRU
+            gru is enforced to be batch_first
+            num_layers default is 1 (number of gru stacked)
+            bidirectional default is False (True not implemented)
+            https://pytorch.org/docs/stable/generated/torch.nn.GRU.html
+
+    The args dicts must respect that:
+        The size of output features in mlp_input must be the same as the input size of the GRU.
+        The size of input features in mlp_output must be the same as the last hidden size of the GRU.
+
+    N = batch size
+    L = sequence size
+    mlp_input_in = input size of the MLP_input
+    mlp_input_out = output size of the MLP_input
+    H_in = input size of the GRU
+    H_out = output size of the GRU
+    mlp_output_in = input size of the MLP_output
+    mlp_output_out = output size of the MLP_output
```

Comment:
What are those fields? I don't see them anywhere

Reply:
Only present in the docstring to document sizes of inputs and outputs.
They would become irrelevant with the suggestion I just made to change the arguments of GRUNet.